.circleci/config.yml

```
@@ -20,7 +28,7 @@ jobs:
             - rush-local-v1-
             - rush-local-v1
       - run:
-          name: Set Up Environment
+          name: Setup CI Environment
```

```suggestion
          name: Set Up Environment
```
I'm not sure why this was changed, but the prior version was grammatically correct and this one isn't.

---

How is this incorrect?  Setup is a word.  The setup being done here is specific to the _CI_ environment.

---

"Setup" is absolutely a word, agreed.

I conflated two things:
1. I find "CI" redundant here because if you're reading this file or anything that references it on the CircleCI website, you _already know it's related to CI_. If there's another environment that this could be confused with, then definitely keep it. It's not a huge deal to me, so if you find the "CI" important here, that's fine!
2. "Setup" is a noun and "Set Up" is a verb. If you want to use "Setup" here, then I think you would need to change the phrasing. Here are a couple of ideas:
    > Setup of CI Environment
    > CI Environment Setup

    This would not match up with the other commands, though, because the ones I looked at start with a verb and are phrased as actions:
    > Check Dependencies
    > Install Dependencies
    > Link Dependencies

---

> "Setup" is a noun and "Set Up" is a verb

I guess so, formally... colloquially the distinction is ... less so

> I find "CI" redundant here

OK ... that is a more useful argument to me.  I find the whole title to be a bit confusing and overly generic -- "Environment" is a very vague term here.  I'm tempted to just re-title this "Initialize ci_scripts" since that's specifically what it is actually doing.

---

I found these answers helpful in satisfying my curiosity on the subject:
- [verbs - Setup or “set up”?](https://english.stackexchange.com/questions/117599/setup-or-set-up)
- [
orthography - login and payoff are nouns. But can they be used as verbs?](https://english.stackexchange.com/questions/93243/login-and-payoff-are-nouns-but-can-they-be-used-as-verbs)

Anyway, yeah, "Initialize ci_scripts" would be fine. Maybe something like "Initialize Tooling" would be a compromise between the two options.